pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -478,8 +478,9 @@ private void startupServiceStatusCheck(long endTimeMs) {
     long checkIntervalMs = _serverConf.getProperty(Server.CONFIG_OF_STARTUP_SERVICE_STATUS_CHECK_INTERVAL_MS,
         Server.DEFAULT_STARTUP_SERVICE_STATUS_CHECK_INTERVAL_MS);
 
+    Status serviceStatus = ServiceStatus.getServiceStatus(_instanceId);
```

Comment:
(minor)
```suggestion
    Status serviceStatus = null;
```

Reply:
I did it this way so we'd have the initial status in log message below if for some reason endTimeMs was immediate. but i guess that seems highly unlikely since it's added to current time, and null is valid in the log